### ISSUE TITLE
Check for is_preview before attempting to use

### ISSUE THREAD

**Issue by seb-b** [takeflight/wagtail-accessibility#4]
I had a page model that didn't use the serve method for rendering. This caused wagtail-accessibility to fail since is_preview was not being set on my request
